var/spack/repos/builtin/packages/py-onnx/package.py

```diff
@@ -39,7 +39,6 @@ class PyOnnx(PythonPackage):
 
     # requirements.txt
     depends_on("py-setuptools@64:", type="build")
```

Comment:
@alalazo the concretizer is selecting setuptools 63 when building py-onnx, which shouldn't be possible: https://gitlab.spack.io/spack/spack/-/jobs/10677665